org.civicrm.bootstrapcivicrm/scss/civicrm/common/_common.scss

```diff
@@ -7,3 +7,11 @@
 @import 'alerts';
 @import 'modals';
 @import 'datepickers';
+@import 'radio-checkbox';
+
+.crm-container {
```

Comment:
Any partial you could put this in? It was nice to have the `_common` file just being a collection of partials